### DISCUSSION TITLE
Whats is the default --cfg while training

### DISCUSSION THREAD

**Discussion by PraveenMNaik** [ultralytics/yolov5#8474] [Q&A]
I wanted to train custom dataset... during training,,,, if my training command doesnt include "--cfg"...... what is the default .yaml file considered....

**Comment by glenn-jocher** [comment]
@PraveenMNaik you can see argparser defaults in train.py:
https://github.com/ultralytics/yolov5/blob/fdc9d9198e0dea90d0536f63b6408b97b1399cc1/train.py#L476

**Reply by PraveenMNaik** [reply]
I got the code... but didn't understand it..... means no ".yaml " file selected?

**Reply by glenn-jocher** [reply]
@PraveenMNaik yes, by default yolov5s.pt is used for --weights and no yaml is selected. See train custom data tutorial for details on defining --weights and --cfg.

## YOLOv5 Tutorials

* [Train Custom Data](https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://github.com/ultralytics/yolov5/wiki/Tips-for-Best-Training-Results)&nbsp; ☘️  RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Roboflow for Datasets, Labeling, and Active Learning](https://github.com/ultralytics/yolov5/issues/4975)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://github.com/ultralytics/yolov5/issues/475)
* [PyTorch Hub](https://github.com/ultralytics/yolov5/issues/36)&nbsp; ⭐ NEW
* [TFLite, ONNX, CoreML, TensorRT Export](https://github.com/ultralytics/yolov5/issues/251) 🚀
* [Test-Time Augmentation (TTA)](https://github.com/ultralytics/yolov5/issues/303)
* [Model Ensembling](https://github.com/ultralytics/yolov5/issues/318)
* [Model Pruning/Sparsity](https://github.com/ultralytics/yolov5/issues/304)
* [Hyperparameter Evolution](https://github.com/ultralytics/yolov5/issues/607)
* [Transfer Learning with Frozen Layers](https://github.com/ultralytics/yolov5/issues/1314)&nbsp; ⭐ NEW
* [Architecture Summary](https://github.com/ultralytics/yolov5/issues/6998)&nbsp; ⭐ NEW

Good luck 🍀 and let us know if you have any other questions!

**Comment by seyeonseanpark** [comment]
With your best.pt, do you still use the --cfg model.yaml file according to the size the model was trained on?
